EpsilonLights/Src/Lights.c

```diff
@@ -294,6 +303,11 @@ void HAL_CAN_RxCpltCallback(CAN_HandleTypeDef* hcan)
         auxBmsInputs[0] = msg->Data[0];
         auxBmsInputs[1] = msg->Data[1];
     }
+    else if (msg->StdId == MOTOR_DRIVE_STDID && msg->DLC == 2)
+    {
+        regenInputs[0] = msg->Data[0];
```

Comment:
Stay consistent on whether you use the `REGENBRAKE_INPUT` macros or not, it's fine either way